resource-managers/kubernetes/integration-tests/pom.xml

```diff
@@ -148,6 +148,9 @@
             <spark.kubernetes.test.master>${spark.kubernetes.test.master}</spark.kubernetes.test.master>
             <spark.kubernetes.test.namespace>${spark.kubernetes.test.namespace}</spark.kubernetes.test.namespace>
             <spark.kubernetes.test.serviceAccountName>${spark.kubernetes.test.serviceAccountName}</spark.kubernetes.test.serviceAccountName>
+            <spark.kubernetes.test.jvmImage>${spark.kubernetes.test.jvmImage}</spark.kubernetes.test.jvmImage>
```

Comment:
So far you can configure the image with `--image-repo` and `--image-tag` but the name part was hardcoded.

Reply:
I know this isn't new, but do all of these have to be system properties vs some command line argument somewhere? Probably, and it's fine, just checking

Reply:
I think we can add corresponding arguments to the `integration-test.sh` script similar to what is supported currently per @skonto's comment.  I will update with that and other requested changes next week

Reply:
@srowen Now also exposed as CLI arguments to the integration test script and documented accordingly in the README